### ISSUE TITLE
Redis triggered template for rust fails

### ISSUE THREAD

**Issue by mikkelhegn** [fermyon/spin#799]
Using the canary build version `spin 0.5.0 (dfb3a90 2022-09-29)` and the template for redis-rust results in the following error on `spin up`:

```
🐟 > spin up
Error: Failed to configure Redis trigger

Caused by:
    metadata error: missing required "redis_address"
Error: exit status: 1
```

**Comment by lann** [comment]
Almost certainly fallout from the big refactor. Looks like an easy fix

**Comment by etehtsea** [comment]
Should be fixed in #800.
